Preparation/Sand_Clock/Program.cs

```diff
@@ -7,17 +8,48 @@ internal class Program
         static void Main(string[] args)
         {
             Print();
-            string x = Console.ReadLine();
-            string y = Console.ReadLine();
 
-            
+            SandCLock();
         }
 
-        static void Print()
+
+        static void SandCLock()
         {
-            Console.WriteLine("Sand Clock");
-            TimeSpan.FromSeconds(10);
-            Console.WriteLine("Enter value: ");
+            int a = int.Parse(Console.ReadLine());
+            int b = int.Parse(Console.ReadLine());
+
+            Console.WriteLine("Enter a size matrix: ");
+            int x = int.Parse(Console.ReadLine());
+
+            int[,] SandMatrix = new int[x,x];
+
+            int smsize = x*x;
+
+            for (int i = 0; i < (x + 1) / 2; i++)
+            {
+                for (int j = i; j < x - i; j++)
+                {
+                    SandMatrix[i, j] = a;
+                    SandMatrix[x - i - 1, j] = a;
+                }
+            }
+            for (int r = 0; r < x && r < x; r++)
+            {
+                for (int c = 0; c < x && c < x; c++)
+                {
+                    Console.Write("{0, 3}", SandMatrix[r, c]);
+                }
+                Console.WriteLine();
+            }
         }
+
+            static void Print()
```

Comment:
Looks like is not needed right now.

But I suggest you divide you code into several logical parts (in main): read input, check input, business logic call (returns result, in this case array int[,]), show output.

This approach would mean a method for each of part, but for now do parameters read and check in main